.gitignore

```diff
@@ -0,0 +1,49 @@
+*.dll
+*.exe
+.DS_Store
+example.tf
+terraform.tfplan
+terraform.tfstate
+bin/
+modules-dev/
+/pkg/
+website/.vagrant
+website/.bundle
+website/build
+website/node_modules
+.vagrant/
+*.backup
+./*.tfstate
+.terraform/
+*.log
+*.bak
+*~
+.*.swp
+.idea
+*.iml
+*.test
+*.iml
+
+website/vendor
+
+.vscode/
+
+# Test exclusions
+!command/test-fixtures/**/*.tfstate
+!command/test-fixtures/**/.terraform/
+
+.env.sh
+
+# goenv version file
+.go-version
+
+# generated by the example acceptance tests
+examples/**/test.tf
+examples/**/test.tfvars
+examples/**/terraform
+examples/**/terraform.zip
+
+#never upload the build to git
+terraform-provider-azurerm
+
+*.ps1
```

Comment:
These should probably be in a global gitignore or feel like they belong to another project, I recommend we remove this
```suggestion
```